### PR TITLE
Some changes to libraries download page

### PR DESCRIPTION
**Pull request by diegoherranz**

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
Diego, the repositories you list are not ready for release, the aim is to have all the data transferred by the v5 software release. Once all the libraries are transferred from the legacy repos then this change can be made :)

**Comment by diegoherranz** [comment]
I know. But this page has always implied they're not ready yet: "In conjunction with the KiCad v5 software release, the libraries will be reorganised into four separate repositories". Maybe it can be made more clear: "[...] four separate repositories which are work in progress" (or similar). Thanks

File: content/libraries/download.adoc [hunk 1]
```diff
@@ -34,11 +36,7 @@ The set of libraries corresponding to `v4.xx` of KiCad software are organised on
 
 In conjunction with the KiCad v5 software release, the libraries will be reorganised into four separate repositories (on GitHub).
```
Comment (SchrodingersGat) [inline content/libraries/download.adoc]: In conjunction with the KiCad v5 software release, the libraries will be reorganised into four separate repositories (on GitHub). These repositories are currently a work in progress.

**Comment by SchrodingersGat** [comment]
Yes I can see what you mean now, I initially misread what you were suggesting. I think overall your changes are valid

**Comment by diegoherranz** [comment]
I've committed the work in progress comment about the v5 repos. Thanks.

**Comment by SchrodingersGat** [comment]
@marekr I am happy with the changes made here

**Pull request merged**